clang-tidy/action.yaml

```diff
@@ -60,9 +60,10 @@ runs:
 
     - name: Run rosdep install
       run: |
+        package_paths=$(colcon list -p --packages-above-and-dependencies ${{ inputs.target-packages }})
```

Comment:
Probably we needed a if-condition here.
https://github.com/tier4/ros2-project-template/runs/5146232233?check_suite_focus=true

Reply:
We should have protected this in `build-and-test.yaml`.

Reply:
Fixed here: https://github.com/autowarefoundation/autoware_common/pull/31